File: app/src/main/java/in/hridayan/ashell/UI/CategoryAbout.java
package in.hridayan.ashell.UI;

public class CategoryAbout {
  private String name;

  public CategoryAbout(String name) {
    this.name = name;
  }

  public String getName() {
    return name;
  }

  public static class LeadDeveloperItem {
    private String title;
    private String description;
    private int imageResource;

    public LeadDeveloperItem(String title, String description, int imageResource) {
      this.title = title;
      this.description = description;
      this.imageResource = imageResource;
    }

    public String getTitle() {
      return title;
    }

    public String getDescription() {
      return description;
    }

    public int getImageResource() {
      return imageResource;
    }
  }

  public static class ContributorsItem {
    private String id, title, description;
    private int imageResource;

    public ContributorsItem(String id, String title, String description, int imageResource) {
      this.id = id;
      this.title = title;
      this.description = description;
      this.imageResource = imageResource;
    }

    public String getId() {
      return id;
    }

    public String getTitle() {
      return title;
    }

    public String getDescription() {
      return description;
    }

    public int getImageResource() {
      return imageResource;
    }
  }

  public static class AppItem {
    private String id, title, description;
    private int imageResource;

    public AppItem(String id, String title, String description, int imageResource) {
      this.id = id;
      this.title = title;
      this.description = description;
      this.imageResource = imageResource;
    }

    public String getId() {
      return id;
    }

    public String getTitle() {
      return title;
    }

    public String getDescription() {
      return description;
    }

    public int getImageResource() {
      return imageResource;
    }
  }
}

File: app/src/main/java/in/hridayan/ashell/ui/CategoryAbout.kt
package `in`.hridayan.ashell.ui

import `in`.hridayan.ashell.config.Const.Contributors

class CategoryAbout(@JvmField val name: String?) {
    class LeadDeveloperItem(
        @JvmField val title: String?,
        @JvmField val description: String?,
        @JvmField val imageResource: Int
    )

    class ContributorsItem(
        id: Contributors?,
        title: String?,
        description: String?,
        imageResource: Int
    ) {
        private val id: Contributors?

        @JvmField
        val title: String?

        @JvmField
        val description: String?

        @JvmField
        val imageResource: Int

        init {
            this.id = id
            this.title = title
            this.description = description
            this.imageResource = imageResource
        }

        fun getId(): Contributors? {
            return id
        }
    }

    class AppItem(
        @JvmField val id: String?,
        @JvmField val title: String?,
        @JvmField val description: String?,
        @JvmField val imageResource: Int
    )
}
